File: 2335. Minimum Amount of Time to Fill Cups.py
#PYTHON
#* 2335. Minimum Amount of Time to Fill Cups
amount = [5,4,4]
from typing import List
from math import ceil
#* FINALLY GOT THIS AHHHH
class Solution:
    def fillCups(self, amount: List[int]) -> int:
        return max(max(amount), ceil(sum(amount)/2)) 

print(Solution().fillCups(amount))



    

File: 2335. Minimum Amount of Time to Fill Cups.rb
#* 2335. Minimum Amount of Time to Fill Cups
amount = [5,4,4]
# @param {Integer[]} amount
# @return {Integer}
#* FINALLY GOT THIS AHHHH
def fill_cups(amount)
  [amount.max, (amount.sum / 2.0).ceil].max
end
p fill_cups(amount)



    